src/components/Requirements/CompletedSubReqCourse.vue

```diff
@@ -100,19 +100,18 @@ export default Vue.extend({
       if (!this.$data.scrollable) event.preventDefault();
     },
     setCourseAndSemesterForNonTransferCredits(crseTaken: CourseTaken) {
+      const courseTakenCode = `${crseTaken.subject} ${crseTaken.code}`;
```

Comment:
CompletedSubReqs seem to show with the default data values. I think `courseTakenCode` should be `${crseTaken.subject} ${crseTaken.number}`
![Screen Shot 2021-02-11 at 11 10 30 AM](https://user-images.githubusercontent.com/54298311/107663787-c2bd3100-6c59-11eb-89da-af5ce5e170e0.png)

Reply:
fixed